### PR TITLE
Add Trino 425 release notes

### LINKED ISSUE #18725: Release notes for 425

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 425 release. Fixes #18725

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

## Verification for each pull request

Format: maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 17 Aug 2023

* #18714 ✅ rn ✅ docs

## 18 Aug 2023

* #18722 ✅ rn ✅ docs
* #18629 ✅ rn ✅ docs
* #18623 ✅ rn ✅ docs
* #18690 ✅ rn ✅ docs
* #18106 ✅ rn ✅ docs

## 19 Aug 2023

* #18737 ✅ rn ✅ docs
* #18683 ✅ rn ✅ docs

## 20 Aug 2023

* #18740 ✅ rn ✅ docs
* #18741 ✅ rn ✅ docs

## 21 Aug 2023

* #18751 ✅ rn ✅ docs
* #18735 ✅ rn ✅ docs
* #18736 ✅ rn ✅ docs
* #18746 ✅ rn ✅ docs
* #18724 ✅ rn ✅ docs
* #16743 ✅ rn ✅ docs
* #18574 ✅ rn ✅ docs
* #18759 ✅ rn ✅ docs
* #18757 ✅ rn ✅ docs

## 22 Aug 2023

* #18756 ✅ rn ✅ docs
* #18773 ✅ rn ✅ docs
* #18774 ✅ rn ✅ docs
* #18768 ✅ rn ✅ docs
* #18320 ✅ rn ✅ docs
* #18770 ✅ rn ✅ docs

## 23 Aug 2023

* #17697 ✅ rn ✅ docs
* #18761 ✅ rn ✅ docs
* #18293 ✅ rn ✅ docs
* #18137 ✅ rn ✅ docs
* #18763 ✅ rn ✅ docs
* #18332 ✅ rn ✅ docs
* #18776 ✅ rn ✅ docs

## 24 Aug 2023

* #18777 ✅ rn ✅ docs
* #18789 ✅ rn ✅ docs
* #18628 ✅ rn ✅ docs
* #18797 ✅ rn ✅ docs
* #17920 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-425.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Release 425 (23 Aug 2023)
+
+## General
+
+* Improve performance of grouping operations. ({issue}`18106`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-425.md:5]: is this about GROUP BY .. or something else more broadly?

File: docs/src/main/sphinx/release/release-425.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Fix incorrect reporting of cumulative memory usage. ({issue}`18714`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-425.md:6]: in the webui and in general I guess?

File: docs/src/main/sphinx/release/release-425.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+* Improve memory usage accounting of the Parquet writer. ({issue}`18756`)
+
+## Kudu connector
+
+* Add support for `CASCADE` option in `DROP SCHEMA` statements. ({issue}`18629`)
+
+## MongoDB connector
+
+* Add support for mapping the `Decimal128` MongoDB type. ({issue}`18722`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-425.md:31]:
```suggestion
* Add support for the `Decimal128` MongoDB type. ({issue}`18722`)
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-425.md (hunk 2 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-425.md:6]: I think general is fine?

File: docs/src/main/sphinx/release/release-425.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## General
+
+* Improve performance of `GROUP BY`. ({issue}`18106`)
+* Fix incorrect reporting of cumulative memory usage. ({issue}`18714`)
+
+## Delta Lake connector
+
+* Add support for check constraints in `MERGE` statements. ({issue}`15411`)
+* Add file-level statistics when statistics are missing from the transaction
```
Comment (martint) [inline docs/src/main/sphinx/release/release-425.md:11]: What does this mean? What's the impact?

**Pull request merged**